gson/src/main/java/com/google/gson/annotations/JsonAdapter.java

```diff
@@ -87,6 +89,7 @@
  * @author Jesse Wilson
  */
 // Note that the above example is taken from AdaptAnnotationTest.
+@Documented
```

Comment:
I was torn on this one. How about Since, Until and Expose? Should we have Documented only for SerializedName?

I am new to Documented, so you can guide me where it is appropriate to be used.